addons/mrp_account/models/mrp_workorder.py

```diff
@@ -50,7 +50,7 @@ def _create_or_update_analytic_entry(self):
             mo_account = wo.production_id.analytic_account_id
             wc_account = wo.workcenter_id.costs_hour_account_id
             if mo_account:
-                is_zero = float_is_zero(value, precision_rounding=mo_account.currency_id.rounding)
+                is_zero = float_is_zero(value, precision_rounding=self.company_id.currency_id.rounding)
```

Comment:
also wc_account

Reply:
Done, Thanks!

Reply:
wo.company_id since it's not always the current company no?